stan/math/prim/mat/prob/bernoulli_logit_glm_rng.hpp

```diff
@@ -0,0 +1,82 @@
+#ifndef STAN_MATH_PRIM_MAT_PROB_BERNOULLI_LOGIT_GLM_RNG_HPP
+#define STAN_MATH_PRIM_MAT_PROB_BERNOULLI_LOGIT_GLM_RNG_HPP
+
+#include <Eigen/Dense>
```

Comment:
Instead of this, we need to be including our own Eigen header which includes the traits in the right way.

```
#include <stan/math/prim/mat/fun/Eigen.hpp>
```